modules/core/src/main/java/org/apache/ignite/internal/visor/tx/VisorTxTask.java

```diff
@@ -221,6 +231,10 @@ private VisorTxJob(VisorTxTaskArg arg, boolean debug) {
             }
 
             for (IgniteInternalTx locTx : transactions) {
+                if (arg.txInfoArgument() != null &&
+                    !arg.txInfoArgument().gridCacheVersion().equals(locTx.nearXidVersion()))
+                    continue;
```

Comment:
Verbose mode on and tx match the criteria, right?

Reply:
Fixed.